packages/components/pie-button/src/button.scss

```diff
@@ -161,18 +216,30 @@
         --btn-padding: 6px var(--dt-spacing-b);
         --btn-font-size: calc(var(--dt-font-size-14) * 1px);
         --btn-line-height: calc(var(--dt-font-size-14-line-height) * 1px);
+        --btn-min-inline-size: 55px;
```

Comment:
@ashleynolan to check these

Reply:
Yes please! I noticed some buttons, even of the same variant, had differing heights (50px vs 55px for example) so would be good to clarify.

Reply:
This shouldn't be an issue now, as the button is the same height as the text. Design will check variant heights in more detail when they go through the design review of the button shortly.